src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -270,8 +270,7 @@ public void launchManageAllergies() {
 
 	@Then("system loads Allergies table page")
 	public void systemLoadsManageAllergies() {
-		assertEquals(getElement(patientHeaderId).getText(),
-				getElement(patientHeaderId).getText());
+				getElement(patientHeaderId).getText();
```

Comment:
Is this change neccessary here

Reply:
> Is this change neccessary here

@sherrif10  it still works fine

Reply:
@Ndacyayisenga-droid  what does `getElement(patientHeaderId).getText()` accomplishing?